ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -28,6 +28,13 @@
 - name: start invoker using docker cli
   shell: >
         docker run -d
+        --pid=host
+        --userns=host
+        --privileged
+        -v /sys/fs/cgroup:/sys/fs/cgroup
+        -v /run/runc:/run/runc
+        -v /usr/bin/docker-runc:/usr/bin/docker-runc
+        -v /usr/lib/x86_64-linux-gnu/libapparmor.so.1:/usr/lib/x86_64-linux-gnu/libapparmor.so.1
```

Comment:
Will this work on *any* system, even if apparmor is not installed (like docker-machine)?